hadoop-hdds/framework/src/main/resources/webapps/static/templates/overview.html

```diff
@@ -14,7 +14,7 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 -->
-<h1>Overview</h1>
+<h1>Overview <small>({{$ctrl.jmx.Hostname}}:{{$ctrl.jmx.HttpPort}})</small> </h1>
```

Comment:
Let's omit the `ctrl.jmx.HttpPort` since technically the page can be accessed using HTTPs.

Reply:
You're right. I will remove `ctrl.jmx.HttpPort`.